cockpit/src/main/java/com/polidea/cockpit/paramsedition/ParamType.kt

```diff
@@ -0,0 +1,21 @@
+package com.polidea.cockpit.paramsedition
+
+import com.polidea.cockpit.core.CockpitParam
+import com.polidea.cockpit.utils.isTypeOf
+
+enum class ParamType {
+    BOOL, INT, DOUBLE, STRING;
+
+    companion object {
+        fun getParamType(param: CockpitParam<Any>) =
+                when {
+                    param.isTypeOf<Boolean>() -> BOOL
+                    param.isTypeOf<Int>() -> INT
+                    param.isTypeOf<Double>() -> DOUBLE
+                    param.isTypeOf<String>() -> STRING
+                    else -> STRING
```

Comment:
Why STRING? Shouldn't it just cause throwing an exception?

Reply:
Well, we can do that. But we can also edit its value as a string. Not sure if throwing an exception is the best way. Let's consider it with Wojtek together.